src/test/java/teammates/test/cases/action/BaseActionTest.java

```diff
@@ -731,4 +731,8 @@ protected void verifyNumberOfEmailsSent(Action action, int emailCount) {
         assertEquals(emailCount, action.getEmailSender().getEmailsSent().size());
     }
 
+    protected String getPageResultDestination(String viewUri, boolean error, String userId) {
+        return viewUri + "?error=" + error + "&user=" + userId;
```

Comment:
Hmm the string "error" and "user" can actually be replaced by `Const.ParamNames.ERROR` and `Const.ParamsNames.USER_ID`.You could also consider using `addParamToUrl` in the `Url` class to append the parameters to the `viewUri`.  
It might be good to rename `viewUri` to something more generic since it might mislead a reader to think it should only be used with constants from `Const.ViewURIs`.